pkg/flist/cleanup.go

```diff
@@ -127,13 +128,18 @@ func (f *flistModule) cleanupAll() error {
 // Cleaner runs forever, checks the tracker files for filesystem processes
 // that requires cleanup
 func (f *flistModule) Cleaner(ctx context.Context, every time.Duration) {
+	slog := log.Sample(zerolog.LevelSampler{InfoSampler: &zerolog.BurstSampler{
```

Comment:
I don't think this one is needed because the slog only sample info, but the only log in this method that uses it is already of type Error.